### PR TITLE
When exporting, hash inputs when an exception happens

### PR DESCRIPTION
**Pull request by henriquegemignani**

### CODE REVIEW

File: randovania/exporter/game_exporter.py [hunk 1]
```diff
@@ -58,18 +67,67 @@ def export_game(
...
+                finally:
+                    self._after_export()
+
+
+def _hash_file_internal(path: Path) -> typing.Any:
+    sha = hashlib.sha1()
+
+    with path.open("rb") as f:
+        while True:
+            data = f.read(65536)
```
Comment (Miepee) [inline randovania/exporter/game_exporter.py:104]: why this number specifically?

File: randovania/exporter/game_exporter.py [hunk 2]
```diff
@@ -58,18 +67,67 @@ def export_game(
...
+                    scope.capture_exception(e, contexts=input_hash)
+                    # TODO: compare `input_hash` with known good values and if different,
+                    # wrap into a "User Error" exception
+                    raise
+
+                finally:
+                    self._after_export()
+
+
+def _hash_file_internal(path: Path) -> typing.Any:
```
Comment (Miepee) [inline randovania/exporter/game_exporter.py:99]: any? :cry:

Comment (henriquegemignani) [reply randovania/exporter/game_exporter.py:99]: I tried but couldn't find a decent hint. Best has `_hashlib.HASH` but at least pycharm didn't like

File: randovania/exporter/game_exporter.py (hunk 1 above)
Comment (henriquegemignani) [reply randovania/exporter/game_exporter.py:104]: random example I found online. Seemed like a decent chunk to read, being 64kb

Comment (Miepee) [reply randovania/exporter/game_exporter.py:104]: If we only read 64kb, isnt the system then useless for e.g the gamecube games where there were dumping errors happening after 64kb?

Comment (henriquegemignani) [reply randovania/exporter/game_exporter.py:104]: There's a `while True`?

Comment (Miepee) [reply randovania/exporter/game_exporter.py:104]: nvm whoops.

**Pull request merged**